DependencyInjection/Configuration.php

```diff
@@ -79,6 +79,7 @@ public function getConfigTreeBuilder()
                 ->scalarNode('title')->end()
                 ->scalarNode('description')->end()
                 ->scalarNode('original_route_pattern')->defaultValue(SeoPresentation::ORIGINAL_URL_CANONICAL)->end()
+                ->scalarNode('locale')->defaultNull()->end()
```

Comment:
Ohh then it would be better to really have the current locale ( in two capital letters as the Country Code).  
Now i remember that change - thought it is a cheap win to do that by the way. 
You can decide what to do with that: completely removing and opening an issue or fixing it by using the request object (be aware of the tests).